packages/x-charts/src/internals/plugins/featurePlugins/useChartCartesianAxis/zoom.types.ts

```diff
@@ -71,6 +71,24 @@ export interface ZoomOptions {
    * @default 'keep'
    */
   filterMode?: ZoomFilterMode;
+  /**
+   * Configures the zoom slider. The slider is an element that shows the zoomed range and allows its manipulation.
+   */
+  slider?: ZoomSliderOptions;
+}
+
+export interface ZoomSliderOptions {
```

Comment:
This probably needs to be exported

Reply:
Hm, that's a good point, but `ZoomOptions` is already being exported from the community package. I think that's a bug in our exports as well. 

I've removed the `export` from `ZoomSliderOptions` since it isn't used outside this file.

Reply:
I've re-exported it from the pro package, then 👍 

Does that make sense? We aren't using this pattern anywhere, but I guess it's fine?